src/main/eo/org/eolang/txt/text.eo

```diff
@@ -60,11 +60,11 @@
   # Checks that string ends with substr
   [substr] > ends-with /bool
 
-  # Checks that string starts with substr
```

Comment:
@levBagryansky why you removed description?

Reply:
@EugeneDar my bad